rosa_tutorials/rosa-mobb-deploy-api-data-protection.adoc

```diff
@@ -0,0 +1,591 @@
+{
+"Version": "2012-10-17",
+"Statement": [
+ {
+   "Effect": "Allow",
+   "Action": [
+     "s3:CreateBucket",
+     "s3:DeleteBucket",
+     "s3:PutBucketTagging",
+     "s3:GetBucketTagging",
+     "s3:PutEncryptionConfiguration",
+     "s3:GetEncryptionConfiguration",
+     "s3:PutLifecycleConfiguration",
+     "s3:GetLifecycleConfiguration",
+     "s3:GetBucketLocation",
+     "s3:ListBucket",
+     "s3:GetObject",
+     "s3:PutObject",
+     "s3:DeleteObject",
+     "s3:ListBucketMultipartUploads",
+     "s3:AbortMultipartUpload",
+     "s3:ListMultipartUploadParts",
+     "ec2:DescribeSnapshots",
+     "ec2:DescribeVolumes",
+     "ec2:DescribeVolumeAttribute",
+     "ec2:DescribeVolumesModifications",
+     "ec2:DescribeVolumeStatus",
+     "ec2:CreateTags",
+     "ec2:CreateVolume",
+     "ec2:CreateSnapshot",
+     "ec2:DeleteSnapshot"
+   ],
+   "Resource": "*"
+ }
+]}
+EOF
+POLICY_ARN=$(aws iam create-policy --policy-name "RosaOadpVer1" \
+--policy-document file:///${SCRATCH}/policy.json --query Policy.Arn \
+--tags Key=rosa_openshift_version,Value=${CLUSTER_VERSION} Key=rosa_role_prefix,Value=ManagedOpenShift Key=operator_namespace,Value=openshift-oadp Key=operator_name,Value=openshift-oadp \
+--output text)
+fi
+echo ${POLICY_ARN}
+----
+
+. Create an IAM Role trust policy for the cluster:
++
+[source,terminal]
+----
+$ cat <<EOF > ${SCRATCH}/trust-policy.json
+{
+  "Version": "2012-10-17",
+  "Statement": [{
+    "Effect": "Allow",
+    "Principal": {
+      "Federated": "arn:aws:iam::${AWS_ACCOUNT_ID}:oidc-provider/${OIDC_ENDPOINT}"
+    },
+    "Action": "sts:AssumeRoleWithWebIdentity",
+    "Condition": {
+      "StringEquals": {
+         "${OIDC_ENDPOINT}:sub": [
+           "system:serviceaccount:openshift-adp:openshift-adp-controller-manager",
+           "system:serviceaccount:openshift-adp:velero"]
+      }
+    }
+  }]
+}
+EOF
+ROLE_ARN=$(aws iam create-role --role-name \
+ "${ROLE_NAME}" \
+  --assume-role-policy-document file://${SCRATCH}/trust-policy.json \
+  --tags Key=rosa_cluster_id,Value=${ROSA_CLUSTER_ID} Key=rosa_openshift_version,Value=${CLUSTER_VERSION} Key=rosa_role_prefix,Value=ManagedOpenShift Key=operator_namespace,Value=openshift-adp Key=operator_name,Value=openshift-oadp \
+  --query Role.Arn --output text)
+
+echo ${ROLE_ARN}
+----
+
+. Attach the IAM Policy to the IAM Role:
++
+[source,terminal]
+----
+$ aws iam attach-role-policy --role-name "${ROLE_NAME}" \
+ --policy-arn ${POLICY_ARN}
+----
+
+== Deploy OADP on the cluster
+
+. Create a namespace for OADP:
++
+[source,terminal]
+----
+$ oc create namespace openshift-adp
+----
+
+. Create a credentials secret:
++
+[source,terminal]
+----
+$ cat <<EOF > ${SCRATCH}/credentials
+[default]
+role_arn = ${ROLE_ARN}
+web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
+EOF
+$ oc -n openshift-adp create secret generic cloud-credentials \
+ --from-file=${SCRATCH}/credentials
+----
+
+. Deploy the OADP Operator:
++
+[NOTE]
+====
```

Comment:
Optional: I think there are some hard returns in the note that can be removed. No big deal either way :)

Reply:
Optional: Double space "status.  This does not seem"